api/service/cart.service.go

```diff
@@ -14,6 +14,7 @@ var (
 type CartService interface {
 	AddItemToCart(ctx context.Context, cartID int64, artToyID int64) (*domain.CartItem, error)
 	CreateCart(ctx context.Context, ownerID int64) (*domain.Cart, error)
+	GetCartByOwnerID(ctx context.Context, ownerID int64) ([]*domain.ArtToy, error)
```

Comment:
```suggestion
	GetCartWithItemByOwnerID(ctx context.Context, ownerID int64) ([]*domain.ArtToy, error)
```
ถ้าแก้ให้มัน Join เขียนเผื่อไว้ด้วยว่าอันนี้ Join แถมมาด้วย